python/ray/tune/examples/async_hyperband_example.py

```diff
@@ -19,10 +19,10 @@ class MyTrainableClass(Trainable):
     maximum reward value reached.
     """
 
-    def _setup(self, config):
+    def build(self, config):
```

Comment:
Is this api finalized already? I was thinking, maybe this could be called `initialize` or `setup`? Not saying those are better than `build`, but just throwing them out here if the names are not yet final.

Reply:
Thanks! none of these names are finalized. Any suggestions welcome..